docs/usage/configuration-options.md

```diff
@@ -1466,8 +1466,9 @@ Note that this limit is enforced on a per-repository basis.
 
 If you configure `prCreation=not-pending`, then Renovate will wait until tests are non-pending (all pass or at least one fails) before creating PRs.
 However there are cases where PRs may remain in pending state forever, e.g. absence of tests or status checks that are configure to pending indefinitely.
-Therefore we configure an upper limit - default 24 hours - for how long we wait until creating a PR.
-Note also this is the same length of time as for Renovate's own `unpublishSafe` status check for npm.
+Therefore we configure an upper limit for how long we wait until creating a PR.
+
+Note: if the option `stabilityDays` is non-zero then Renovate will disable `prNotPendingHours` functionality.
```

Comment:
```suggestion
Note: if the option `stabilityDays` is non-zero then Renovate will disable the `prNotPendingHours` functionality.
```

Reply:
@HonkingGoose you're the best goose I ever knew